app/scripts/importUiMetaData/metaData.js

```diff
@@ -2118,7 +2118,7 @@ export default {
     },
     itemName: {
       itemNameTranslationKey: 'arrays.productGroupsItemName',
-      itemNameField: 'group_id',
+      itemNameField: 'summary',
```

Comment:
As discussed, that change should be reverted.
```suggestion
      itemNameField: 'group_id',